### ISSUE TITLE
confusing names in ampl_ipopt.cpp

### ISSUE THREAD

**Issue by svigerske** [coin-or/Ipopt#172]
Issue created by migration from Trac.

Original creator: baharev

Original creation time: 2011-10-26 10:00:47

Assignee: ipopt-team

Version: 3.10

In ampl_ipopt.cpp, line 34, instead of

```
int main(int argv, char**argc)
```

I suggest

```
int main(int argc, char** argv)
```

Swapping the conventional names argc and argv is confusing.

See also [http://en.wikipedia.org/wiki/Main_function#C_and_C.2B.2B]

**Comment by svigerske** [comment]
Comment by @svigerske created at 2012-02-16 12:57:06

Fixed in trunk rev2092.

**Comment by svigerske** [comment]
Comment by @svigerske created at 2012-11-19 15:49:28

Merged to stable/3.10 and will be in next release.

**Comment by svigerske** [comment]
Comment by @svigerske created at 2012-11-19 15:49:28

Resolution: fixed
